airflow/example_dags/example_skip_dag.py

```diff
@@ -30,21 +32,31 @@
 
 # Create some placeholder operators
 class DummySkipOperator(DummyOperator):
+    """Dummy operator which always skips the task."""
+
     ui_color = '#e8b7e4'
 
     def execute(self, context):
         raise AirflowSkipException
 
 
-def create_test_pipeline(suffix, trigger_rule, dag):
-    skip_operator = DummySkipOperator(task_id='skip_operator_{}'.format(suffix), dag=dag)
-    always_true = DummyOperator(task_id='always_true_{}'.format(suffix), dag=dag)
-    join = DummyOperator(task_id=trigger_rule, dag=dag, trigger_rule=trigger_rule)
-    final = DummyOperator(task_id='final_{}'.format(suffix), dag=dag)
+def create_test_pipeline(suffix, trigger_rule, dag_):
+    """
+    Instantiate a number of operators for the given DAG.
```

Comment:
no need empty first line

Reply:
Empty line is required after method's description.

Reply:
I remember you telling my about this yes, forgot once again.

I tried searching for the exact problem but couldn't find it, I thought you said the docs don't get rendered correctly without the whitespace? E.g. the BashOperator.execute() pydoc is written without whitespace but the docs look okay: https://airflow.apache.org/_api/airflow/operators/bash_operator/index.html#airflow.operators.bash_operator.BashOperator.execute.

Could you let me know the exact issue?

Reply:
It's contains the empty line after description. 
<img width="727" alt="Screenshot 2019-06-08 at 22 07 12" src="https://user-images.githubusercontent.com/12058428/59151832-2b421c80-8a3a-11e9-90a7-b8a1581ab013.png">

I do not know why it is required, but the documentation without this line is not correctly rendered.
I made a small change in http_operator.py to demonstrate problem.
<img width="752" alt="Screenshot 2019-06-08 at 22 21 10" src="https://user-images.githubusercontent.com/12058428/59151959-c4bdfe00-8a3b-11e9-9f6c-38bb984b71bd.png">
I receive a error message:
```
/app/airflow/docs/_api/airflow/operators/http_operator/index.rst:36: WARNING: Unexpected indentation.
/app/airflow/docs/_api/airflow/operators/http_operator/index.rst:37: WARNING: Block quote ends without a blank line; unexpected unindent.
```

Reply:
Original content is rendered correctly:
<img width="575" alt="Screenshot 2019-06-08 at 22 22 22" src="https://user-images.githubusercontent.com/12058428/59151971-ed45f800-8a3b-11e9-9a0f-b46933c7d346.png">

Reply:
Ah my bad, I interpreted it wrong... (was thinking of whitespace after the pydoc instead of description...)